package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -89,12 +89,14 @@ define([
     },
 
     /**
-     * Orchestrates the rendering of the visualization.
+     * Orchestrates the rendering of the visualization and is meant to be invoked by the container.
      *
-     * Executes the will/did/rejected event loop associated with an update in a visualization.
+     * Executes [_update]{@link pentaho.visual.base.View#_update} asynchronously in
+     * the will/did/rejected event loop associated with an update of a visualization,
+     * also creating the visualization DOM element the first time it successfully updates.
```

Comment:
and also creates the visualization DOM element the first time it successfully updates.